src/ensembl/src/content/app/entity-viewer/gene-view/styles/_constants.scss

```diff
@@ -1,5 +1,13 @@
+$gene_image_width: 695;
```

Comment:
To avoid multiplying by one pixel in other scss files (which looks odd there) we should probably confine all the ugliness just to this file, by doing something like:

```scss
$gene_image_width_number: 695;
$gene_image_width: #{695}px;

...

:export {
  gene_image_width: $gene_image_width_number;
}
```

then, I hope, you'll be able to remove `* 1px` from the scss files that are using the `$gene_image_width` variable.

Reply:
Agree about having it in one place. The reason I changed it to this is because I read about sting interpolation in scss is a bad idea and it has a powerful unit arithmetic that you can use instead  https://sass-lang.com/documentation/interpolation

Reply:
You can do the multiplication if you prefer, but why is the interpolation a bad idea? I haven't heard about this.